bin/build_and_push.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/sh
+REGION=us-east-2
+ENVIRONMENT=${1:-dev}
+ECR_URL=377928551571.dkr.ecr.us-east-2.amazonaws.com/relayer-$ENVIRONMENT
+
+GIT_HASH=$(git rev-parse HEAD)
+docker build -t relayer:latest  -f ./docker/release/Dockerfile .
```

Comment:
should we have the build target `--platform linux/arm64`? maybe that's why the local build is noticeably faster over the BuildJet runners, which do this?

Reply:
dockerfile already defaults to this